src/application.rs

```diff
@@ -158,7 +158,7 @@ impl Application {
             .transient_for(&self.main_window())
             .modal(true)
             .authors(vec!["Marco Melorio".into(), "Marcus Behrendt".into()])
-            .artists(vec!["Marco Melorio".into()])
+            .artists(vec!["Marco Melorio".into(), "Mateus Santos".into() ,"noëlle".into()])
```

Comment:
```suggestion
            .artists(vec!["Marco Melorio".into(), "Mateus Santos".into() , "noëlle".into()])
```
Nitpick whitespace fix. Also, is your name deliberately in all lowercase?

Reply:
Rustfmt also complains.